File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = phone_number;
    
    for(int i = 0; i < phone_number.length() - 4; i++) {
        answer = phone_number.replace(i, 1, "*");
    }
    return answer;
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cs
public class Solution {
    public string solution(string phone_number) {
        string answer = "";
        string str = phone_number.Substring(phone_number.Length - 4);
        for(int i = 0; i < phone_number.Length-4; i++) {
            answer += "*";
        }
        
        answer += str;
        return answer;
    }
}